README.md

```diff
@@ -1,12 +1,11 @@
-Python and Astropy for Astronomical Data Analysis
-=================================================
-*Workshop at the 245th Meeting of the AAS in National Harbor, MD, USA*
+# Astropy Learning at Observatories of HAwaii (ALOHA)
 
-* **DATE:** Sunday January 12, 2025
-* **TIME:** 9:00 am – 5:30 pm
-* **LOCATION:** TBD
+*Workshops at Subaru and Keck*
 
-## PRE-WORKSHOP SETUP
+
+## PRE-WORKSHOP TO-DOs
+
+### Install Python if you do not have it
```

Comment:
I cannot get to L16 but I do not think it is accurate for this workshop?

`If you have any trouble, we will have facilitators on-site as early as 8:30 AM local time who can help you in person.`

Reply:
Correct, that is not accurate, fixing